tdrs-frontend/src/components/SubmissionHistory/SubmissionHistory.jsx

```diff
@@ -68,17 +228,33 @@ const SectionSubmissionHistory = ({ section, label, files }) => {
   const pageEnd = Math.min(files.length, pageStart + pageSize)
 
   return (
-    <div className="submission-history-section">
+    <div
+      className="submission-history-section usa-table-container--scrollable"
+      style={{ maxWidth: '1400px', margin: '100px' }}
```

Comment:
The max width here depends on the size of screen, so I suggest going with % instead of absolute value